src/main/java/se/sundsvall/supportmanagement/api/validation/impl/ValidTypeTagConstraintValidator.java

```diff
@@ -1,22 +1,31 @@
 package se.sundsvall.supportmanagement.api.validation.impl;
 
+import static java.util.Collections.emptyList;
+import static java.util.Optional.ofNullable;
+
+import java.util.List;
+
 import javax.validation.ConstraintValidator;
 import javax.validation.ConstraintValidatorContext;
 
 import org.springframework.beans.factory.annotation.Autowired;
 
+import se.sundsvall.supportmanagement.api.model.metadata.Type;
 import se.sundsvall.supportmanagement.api.validation.ValidTypeTag;
-import se.sundsvall.supportmanagement.service.TagService;
+import se.sundsvall.supportmanagement.service.MetadataService;
 
 public class ValidTypeTagConstraintValidator extends AbstractTagConstraintValidator implements ConstraintValidator<ValidTypeTag, String> {
```

Comment:
Remove tag-suffix?

Reply:
Same comment as in ValidStatusTagConstraintValidator